### PR TITLE
UI Snapshot Test - Verify clicking on VideoTile ellipses opens contextual menu in Desktop and test Pin/Unpin

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
UI Snapshot Test - Verify clicking on VideoTile ellipses opens contextual menu in Desktop and test Pin/Unpin

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3021647

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by alkwa-msft — APPROVED** [review]

File: packages/react-composites/tests/browser/call/hermetic/VideoGalleryLayout.test.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  test
+} from './fixture';
+import { expect } from '@playwright/test';
+import { dataUiId, waitForSelector, stableScreenshot, isTestProfileMobile, pageClick } from '../../common/utils';
+import { IDS } from '../../common/constants';
+
+test.describe('VideoGalleryLayout tests', async () => {
+  /* @conditional-compile-remove(pinned-participants) */
+  test('VideoTile pin/unpin the remote participant', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileMobile(testInfo));
```
Comment (alkwa-msft) [inline packages/react-composites/tests/browser/call/hermetic/VideoGalleryLayout.test.ts:18]: its not clear from this test that its only meant for desktop besides ignoring this for mobile. 

Should there be a mobile test as well?

Comment (prabhjot-msft) [reply packages/react-composites/tests/browser/call/hermetic/VideoGalleryLayout.test.ts:18]: yes, there are separate tests for mobile drawer menu on long press. Modified the test title to make it clear that it is only for desktop

File: packages/communication-react/review/stable/communication-react.api.md [hunk 2]
```diff
@@ -1730,6 +1730,7 @@ export interface _Identifiers {
     typingIndicator: string;
     videoGallery: string;
     videoTile: string;
+    videoTileMoreOptionsButton: string;
```
Comment (dmceachernmsft) [inline packages/communication-react/review/stable/communication-react.api.md:1733]: This probably shouldn't be in the API file here. is there some missing CC?

File: packages/react-components/src/components/VideoTile.tsx [hunk 3]
```diff
@@ -192,12 +192,14 @@ const videoTileMoreMenuProps = {
 /* @conditional-compile-remove(pinned-participants) */
 const VideoTileMoreOptionsButton = (props: { contextualMenu?: IContextualMenuProps }): JSX.Element => {
   const { contextualMenu } = props;
+  const ids = useIdentifiers();
```
Comment (dmceachernmsft) [inline packages/react-components/src/components/VideoTile.tsx:195]: is this a new hook that provides us with the data-ui-ids? is this context wrapping the composite internally? I haven't seen this yet? very cool!

Comment (mgamis-msft) [reply packages/react-components/src/components/VideoTile.tsx:195]: It is a context wrapping the composite internally and has been in our code for a while :). `_IdentifierProvider` is internal so I think it is ok to assign data-ui-id props directly as a prop to components but we should have a team discussion about it

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**